packages/react/src/primitives/Alert/Alert.tsx

```diff
@@ -73,6 +78,7 @@ const AlertPrimitive: Primitive<AlertProps, typeof Flex> = (
 /**
  * [📖 Docs](https://ui.docs.amplify.aws/react/components/alert)
  */
-export const Alert = React.forwardRef(AlertPrimitive);
+export const Alert: ForwardRefPrimitive<BaseAlertProps, 'div'> =
+  React.forwardRef(AlertPrimitive);
```

Comment:
Think adding the type annotation here (as well as for the rest of the primitives) is extraneous since `React.forwardRef` will infer the generics from the `AlertPrimitive` function signature?
```suggestion
export const Alert = React.forwardRef(AlertPrimitive);
```

Reply:
Makes sense, thanks for the detailed explanation!